samples/csharp/sharedcontent/console/speech_recognition_samples.cs

```diff
@@ -1263,13 +1354,25 @@ public static async Task PronunciationAssessmentContinuousWithFile()
                     // Waits for completion.
                     await recognizer.StopContinuousRecognitionAsync().ConfigureAwait(false);
 
+                    // set the duration of Word in pronWords
+                    pronWords.Zip(durations, (word, duration) => word.Duration = duration).ToList();
+
                     // For continuous pronunciation assessment mode, the service won't return the words with `Insertion` or `Omission`
                     // even if miscue is enabled.
                     // We need to compare with the reference text after received all recognized words to get these error words.
-                    string[] referenceWords = referenceText.ToLower().Split(' ');
-                    for (int j = 0; j < referenceWords.Length; j++)
+                    string[] referenceWords;
+
+                    if (language == "zh-CN")
                     {
-                        referenceWords[j] = Regex.Replace(referenceWords[j], "^[\\p{P}\\s]+|[\\p{P}\\s]+$", "");
+                        referenceWords = GetReferenceWords(waveFileName, referenceText, language, config).ToArray();
```

Comment:
here should use a very short audio like Yan did in the python sample code.

Reply:
done, update with short audio